src/app/booking/manage/add/page.tsx

```diff
@@ -36,18 +36,19 @@ export default function BookingPage() {
               throw new Error("Failed to submit create Appointment form");
             }
 
-            const responseData: ResponseData = await response.json();
+            // const responseData:ResponseData = await response.json();
```

Comment:
what's the commenting for?

Reply:
This part will be implemented later by those who will handle the implementation part . For now, we'll comment on it, and you can uncomment it if needed.